kubernetes-dev-environment.yaml

```diff
@@ -14,8 +14,8 @@ spec:
     spec:
       restartPolicy: Never
       containers:
-        - name: dev-environment
-          image: 896129387501.dkr.ecr.us-west-2.amazonaws.com/allennlp:dev-environment
+        - name: allennlp-dev-env
+          image: 896129387501.dkr.ecr.us-west-2.amazonaws.com/allennlp/allennlp-gpu:latest
```

Comment:
if there's a script that's really only intended for internal ai2 users it should not live in the public-facing `allennlp` repo. not sure what the right place for it is though.